### DISCUSSION TITLE
Conditionally display image.

### DISCUSSION THREAD

**Discussion by saurabh-lohiya** [fastn-stack/fastn#652] [Q&A] [answered]
Can we show or hide ftd.image on a if condition? Like following
-- ftd.image if { ftd.device == "desktop" }: $dashboard.src
Show the image only when the device is desktop

**Comment by gsalunke** [comment, marked as answer]
@saurabh-lohiya `ftd.image` can be used with `if` condition as shown below:

For desktop:

```ftd
-- ftd.image:
if: { ftd.device != "mobile" }
src: $dashboard.src
```

For mobile:

```ftd

-- ftd.image:
if: { ftd.device == "mobile" }
src: $dashboard.src

```

**Comment by amitu** [comment]
@saurabh-lohiya this should work:

```ftd
-- ftd.image: 
if: { ftd.device == "desktop" }
src: $dashboard.src
```
